File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromAlphaToProductionTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.PublishingService;

public class PromoteFromAlphaToProductionTask extends BaseTask {
	
	public PromoteFromAlphaToProductionTask() {
		setDescription("Promote a current alpha to production");
	}

	@Override
	protected void onExecute(App app) {
		new PublishingService().promoteFromAlphaToProduction(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromAlphaToProductionTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class PromoteFromAlphaToProductionTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().promoteFromAlphaToProduction(app)
    }

    init {
        description = "Promote a current alpha to production"
    }
}
